server/src/test/java/io/druid/segment/loading/LocalDataSegmentPusherTest.java

```diff
@@ -48,23 +51,23 @@
   LocalDataSegmentPusherConfig config;
   File dataSegmentFiles;
   DataSegment dataSegment = new DataSegment(
-      "ds",
-      new Interval(0, 1),
-      "v1",
-      null,
-      null,
-      null,
-      NoneShardSpec.instance(),
-      null,
-      -1
+    "ds",
+    new Interval(0, 1),
+    "v1",
+    null,
+    null,
+    null,
+    NoneShardSpec.instance(),
+    null,
+    -1
   );
 
   @Before
   public void setUp() throws IOException
   {
     config = new LocalDataSegmentPusherConfig();
     config.storageDirectory = temporaryFolder.newFolder();
-    localDataSegmentPusher = new LocalDataSegmentPusher(config, new ObjectMapper());
+    localDataSegmentPusher = new LocalDataSegmentPusher(config, new JodaMapper());
```

Comment:
For testing purposes consistency and predictability is more important than speed. Since `DefaultObjectMapper` is the de-facto standard `ObjectMapper` for druid code, we should use that whenever possible.